src/graphdb.js

```diff
@@ -61,33 +80,48 @@ function graphDBInferenceSchema (graphbSchema, addMutations) {
 
     gdbs.nodeStructures.forEach(node => {
         // node type
-        let nodeCase = node.label;
+        let nodeCase = replaceCleanseLabel(node.label);
         if (changeCase) {
-            nodeCase = toPascalCase(node.label); 
+            nodeCase = toPascalCase(nodeCase);
+        }
+
+        if (node.label !== nodeCase) {
             r += `type ${nodeCase} @alias(property:"${node.label}") {\n`;
-        } else {
+        }
+        else {
             r += `type ${nodeCase} {\n`;
         }
         
         r += '\t_id: ID! @id\n';
 
         node.properties.forEach(property => {
-            if (property.name == 'id')
+            if (property.name == 'id') {
                 r+= `\tid: ID\n`;
-            else
-                r+= `\t${property.name}: ${property.type}\n`;
+            }
+            else {
+                let propertyCase = replaceCleanseLabel(property.name);
+                let alias = '';
+                if (property.name !== propertyCase) {
+                    alias = ` @alias(property: "${property.name}")`;
+                }
+                r+= `\t${propertyCase}: ${property.type}${alias}\n`;
+            }
         });
         
         let edgeTypes = [];
         gdbs.edgeStructures.forEach(edge => {            
             edge.directions.forEach(direction => {
-                let fromCase = toPascalCase(direction.from);
-                let toCase = toPascalCase(direction.to);
-                let edgeCase = toPascalCase(edge.label);
+                let fromCase = replaceCleanseLabel(direction.from);
+                fromCase = toPascalCase(fromCase);
+                let toCase = replaceCleanseLabel(direction.to);
+                toCase = toPascalCase(toCase);
+                let edgeCase = replaceCleanseLabel(edge.label);
+                edgeCase = toPascalCase(edgeCase);
+
                 if (direction.from == node.label && direction.to == node.label){
                     if (direction.relationship == 'MANY-MANY') {
-                        r += `\t${nodeCase.toLocaleLowerCase() + edgeCase}sOut(filter: ${nodeCase}Input, options: Options): [${nodeCase}] @relationship(edgeType:"${edge.label}", direction:OUT)\n`;
-                        r += `\t${nodeCase.toLocaleLowerCase() + edgeCase}sIn(filter: ${nodeCase}Input, options: Options): [${nodeCase}] @relationship(edgeType:"${edge.label}", direction:IN)\n`;                    
+                        r += `\t${nodeCase.toLocaleLowerCase() + edgeCase}sOut(filter: ${nodeCase} Input, options: Options): [${nodeCase}] @relationship(edgeType:"${edge.label}", direction:OUT)\n`;
```

Comment:
Somehow a space was added before all `Input` which is breaking the schema validity.